cnf-certification-test/accesscontrol/namespace/namespace.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
```

Comment:
2022?

Reply:
fixed. I am not sure how this works. I read this page (https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code). Seems like it is better to have one date instead of a range an not touch it unless something significant changes. Maybe we should discuss our rule and add it to the developing guide?

Reply:
From what I know about copyrights in code, it's supposed to be like when the code was first committed.  And whenever you "touch" it, make sure the year is updated to the latest year.  We'll have to figure it out.

Reply:
fixed. Note sure what is the rule we follow, maybe we could document it in the dev guide? Found this : https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code